lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -31,6 +31,8 @@ class BarChartData extends AxisChartData with EquatableMixin {
   /// You can annotate some regions with a highlight color using [rangeAnnotations].
   ///
   /// You can modify [barTouchData] to customize touch behaviors and responses.
+  ///
+  /// Horizontal lines are drawn with [extraLinesData]. Vertical lines will not be painted if received.
```

Comment:
Can you please explain more about why vertical lines are not supported?
We can write that it is a technical issue that we are trying to solve it, and you can mention the #1149 issue.

Reply:
It was about not exposing functionality to the user that was unintuitive since the user can't pass in a number for the X axis and expect it to appear where they want it.

However, it is easy enough to enable now and allow the issue you mentioned drive the solution. I will change the code to allow vertical lines and mention the issue.

Reply:
Oh sorry, it seems that I didn't explain clearly and there is a miscommunication here.
Actually, I wanted to say that there is no problem that you didn't implement the vertical lines on the BarChart, I just wanted to ask you to explain it and mention the issue (to let people follow it up).

Reply:
Ah, not a problem. Sorry about that. :) Would you like me to revert the code changes to enable vertical lines and add extra comments why it isn't enabled yet? I'm okay with this if that is what you want. Just let me know.

Reply:
Yes in my opinion it is better to revert your code and just add some extra comments. (exactly what you said).
Thanks a lot.

Reply:
Okay, got that updated. I like this better too.